src/app/components/ModalContainer/index.tsx

```
@@ -4,50 +4,50 @@ import { ConvertPokemonUnionSpeciesType } from "@/app/type/pokemon";
+			<div className="fixed inset-0 overflow-y-auto">
+				<div className="flex items-center justify-center min-h-screen p-4">
+					<div
+						className="relative w-full max-w-2xl border shadow-lg bg-white/80 backdrop-blur-lg rounded-xl border-white/20"
+						onClick={(e) => e.stopPropagation()}
+					>
+						<div className="absolute top-4 right-4">
+							<button
+								onClick={handleModelClose}
+								className="text-gray-500 transition-colors hover:text-gray-700"
+								aria-label="Close modal"
+							>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)